pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -236,6 +243,76 @@ public void processSqlWithMultiStageQueryEnginePost(String query, @Suspended Asy
     }
   }
 
+  @POST
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("query/compare")
+  @ApiOperation(value = "Query Pinot using both the single stage query engine and the multi stage query engine and "
```

Comment:
I am using swagger APIs quite a bit today. Setting `sql` vs `v1Sql/v2Sql` maybe confusing. Majority of users will only use the doc in the swagger page to use the API. There are a couple of options to reduce confusion:
* Only provide `v1Sql`/`v2Sql`. Its not that hard to copy paste twice.
* Change the one-line documentation to add more info. There is precedence to multi-line description in swagger page. For example:

> Query Pinot using both the single stage query engine and the multi stage query engine and compare the results. Set sql field to run the same query in both engines. Set v1Sql & v2Sql instead if query text is different.

Reply:
Good point, I've gone with your option 2.